.github/workflows/bundle.yaml

```diff
@@ -25,11 +28,13 @@ jobs:
         else
           echo BUNDLE_TAG="$(git describe --tags --always)" >> $GITHUB_ENV
         fi
-        echo BUNDLE="quay.io/operator-framework/combo-bundle:${BUNDLE_TAG}" >> $GITHUB_ENV
+        echo BUNDLE_REPO="quay.io/operator-framework/combo-bundle" >> $GITHUB_ENV
```

Comment:
When printing out the $BUNDLE variable in a subsequent step, the tag value was empty, so we changed this variable to reflect the repository name, and concatenated the repo:tag value in the docker/build-push-action step.